airbyte-integrations/connectors/source-postgres/src/test-integration/java/io/airbyte/integrations/io/airbyte/integration_tests/sources/AbstractPostgresSourceAcceptanceTest.java

```diff
@@ -0,0 +1,18 @@
+package io.airbyte.integrations.io.airbyte.integration_tests.sources;
+
+import io.airbyte.integrations.base.ssh.SshHelpers;
+import io.airbyte.integrations.standardtest.source.SourceAcceptanceTest;
+import io.airbyte.protocol.models.v0.ConnectorSpecification;
+import java.util.Optional;
+
+public abstract class AbstractPostgresSourceAcceptanceTest extends SourceAcceptanceTest {
```

Comment:
Created this abstract class so that I didn't need to repeat the `"security"` group string in every test